rfc6962-bis.xml

```diff
@@ -930,7 +936,7 @@ an accepted root certificate.
             </list></t>
           </list>
 
-          Both tree sizes must be from existing v1 STHs (Signed Tree Heads).
+          Both tree sizes must be from existing v1 STHs (Signed Tree Heads). However, because of skew, the receiving server may not know one or both of the existing STHs. If both are known, then only the <spanx style="verb">consistency</spanx> output is returned. If the first is known but not the second, then the latest known STH is returned, along with a consistency proof between the first STH and the latest.
```

Comment:
Actually, I was planning to have an error in that case, but on second thoughts returning the STH makes more sense.

I have switched to front-end throughout.